baselines/flanders/plotting/plots.ipynb

```diff
@@ -0,0 +1,636 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 166,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import libraries\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "\n",
+    "from matplotlib import pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 182,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def trim_attack_fn(df):\n",
+    "    \"\"\"In column \"attack_fn\" of dataframe df, remove all\n",
+    "    superfluous characters from the attack name, leaving only\n",
+    "    the attack name itself.\n",
+    "    \"\"\"\n",
+    "    names = [\"gaussian_attack\", \"lie_attack\", \"fang_attack\", \"minmax_attack\"]\n",
+    "    # remove superfluous characters from attack names in df\n",
+    "    for name in names:\n",
+    "        df.loc[df[\"attack_fn\"].str.contains(name), \"attack_fn\"] = name\n",
+    "    names = {\"gaussian_attack\": \"GAUSS\", \"lie_attack\": \"LIE\", \"fang_attack\": \"OPT\", \"minmax_attack\": \"AGR-MM\"}\n",
```

Comment:
The generated `outputrs/all_results.csv` doesn't contain those keys with the `_attack` suffix. I believe that line should be replaced with:
```
    names = {"gaussian": "GAUSS", "lie": "LIE", "fang": "OPT", "minmax": "AGR-MM"}
```